extensions/Worker.Extensions.Http/src/FromBodyConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.Converters
+{
+    internal class FromBodyConverter : IInputConverter
+    {
+        public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
+        {
+            var bodyConversionFeature = context.FunctionContext.Features.Get<IFromBodyConversionFeature>()
+                 ?? DefaultFromBodyConversionFeature.Instance;
```

Comment:
Not 100% happy with the default not being present in the context and used as a fallback. It enables the behavior, but does mean the feature is only present when dealing with the non-default behavior.

I have some changes planned to address this (same pattern of already in place in different flows), but not tracking that for this PR